ocaml/xapi/xapi_network_sriov.mli

```diff
@@ -0,0 +1,27 @@
+(*
```

Comment:
Does this work at all without a xapi_network_sriov.ml file?

Reply:
It should raise some error at runtime but I think we shouldn't care about it at this stage because it's not a complete feature yet.
For now it's aim to make compiler not to complain the lack of the definition of the function that we used so we can pass the Travis check.
P.S. I separate the *Datamodel definition* and *Network sriov object ceate/destroy* into different PRs to make them easier for review.